js/src/components/schema/ColumnNameCell.tsx

```diff
@@ -37,6 +39,8 @@ export function ColumnNameCell({
   const lineageViewContext = useLineageViewContext();
   const { isActionAvailable } = useLineageGraphContext();
   const columnType = currentType ?? baseType;
+  const [cllRunning, setCllRunning] = useState(false);
+  const [cllParams, setCllParams] = useState<CllParams>();
```

Comment:
This state is not necessary here, because the state is kept in the column-specific component